app/src/main/java/org/jellyfin/androidtv/util/sdk/BaseItemExtensions.kt

```diff
@@ -62,6 +62,20 @@ fun BaseItemDto.getProgramSubText(context: Context) = buildString {
 	// Add the channel name if set
 	channelName?.let { append(channelName, " - ") }
 
+	// Add season and episode info if available
+	if (parentIndexNumber != null && parentIndexNumber != 0) {
+		append(context.getString(R.string.lbl_season_number, parentIndexNumber))
+		append(" ")
+	}
+	if (indexNumber != null) {
+		if (indexNumberEnd != null) {
+			append(context.getString(R.string.lbl_episode_range, indexNumber, indexNumberEnd))
+		} else {
+			append(context.getString(R.string.lbl_episode_number, indexNumber))
+		}
+		append(" - ")
+	}
```

Comment:
Similar to getFullName for EPISODE:

```suggestion
	}

	if (indexNumber != null && indexNumberEnd != null)
		append(context.getString(R.string.lbl_episode_range, indexNumber, indexNumberEnd), " - ")
	else if (indexNumber != null)
		append(context.getString(R.string.lbl_episode_number, indexNumber), " - ")
```